src/app/components/users/controllers/show.js

```diff
@@ -0,0 +1,23 @@
+angular.module('librarium')
+  .controller('UsersShowController',
+    ['UsersService', '$stateParams', 'CurrentUserService',
+    function (UsersService, $stateParams, CurrentUserService) {
+      const self = this;
+      self.profileId = $stateParams.id;
+
+      self.userProfile = UsersService.getUser(self.profileId)
```

Comment:
Query this user in a [resolve block](https://github.com/angular-ui/ui-router/wiki#resolve) in the route